docs/en/latest/FAQ.md

```diff
@@ -318,7 +318,7 @@ The high availability of APISIX can be divided into two parts:
 
 ## Why does the `make deps` command fail in source installation?
 
-When executing the `make deps` command, an error such as the one shown below occurs. This is caused by the missing openresty's `openssl` development kit, you need to install it first. Please refer to the [install-dependencies.md](install-dependencies.md) document for installation.
+When executing the `make deps` command, an error such as the one shown below occurs. This is caused by the missing openresty's `openssl` development kit, you need to install it first. Please refer to the [install-dependencies.md](https://github.com/apache/apisix/blob/master/docs/en/latest/install-dependencies.md) document for installation.
```

Comment:
Look like we need to add `install-dependencies` to `latest/config.json`?
CC @juzhiyuan

Reply:
just checked, yea, we should

Reply:
Yes . I will fix this

Reply:
Please solve this so that I can merge it

Reply:
No I think it will work fine without adding  install-dependencies to latest/config.json  as same as how to build doc.
![1](https://user-images.githubusercontent.com/53715187/115070003-00685f80-9f12-11eb-95f9-73d91de00d90.PNG)
![2](https://user-images.githubusercontent.com/53715187/115070011-02cab980-9f12-11eb-9a53-280df4541135.PNG)

Reply:
Earlier, the issue  we had, is  install-dependencies.**_md_**(install-dependencies.md) instead of  install dependencies(install-dependencies.md)

Reply:
If we don't add them to config.json, we could render it, but without the left sidebar actually 😂

Reply:
https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages
![99](https://user-images.githubusercontent.com/53715187/115101249-13098580-9f60-11eb-9142-2b7b96aaabaf.PNG)
![999](https://user-images.githubusercontent.com/53715187/115101255-16047600-9f60-11eb-8aa5-a4cc8fcc6e82.PNG)

Reply:
We mean the left sidebar of our website. The `config.json` is the index.